CHANGELOG.md

```diff
@@ -1,7 +1,29 @@
 Changelog
 =========
 
-## vNext
+## 0.7.0
```

Comment:
Please reference your PR somewhere within the 0.7.0 section (see other versions / references below)